channeldb/channel.go

```diff
@@ -1346,6 +1346,41 @@ func (c *OpenChannel) MarkScidAliasNegotiated() error {
 	return nil
 }
 
+// MarkZeroConfNegotiated adds ZeroConfBit to ChanType in-memory and in the
+// database. If the channel does not yet exist in the database then only the
+// in-memory version is updated.
+func (c *OpenChannel) MarkZeroConfNegotiated() error {
```

Comment:
might not need the DB part... cause I think we would never persist before completing the processing of AcceptChannel right?